Automattic/woocommerce-payments
# Fee Display Causing Confusion for Merchants

Fixes #3666

#### Changes proposed in this Pull Request
There were cases when users got confused with high fees. This PR solves that by letting user know that fees start from a specific value (base rate) instead of displaying total possible transaction fee.


**Before**
![image](https://user-images.githubusercontent.com/15913573/157273615-017883d5-3539-4c01-b0eb-746947a01fa6.png)


**After**
![image](https://user-images.githubusercontent.com/15913573/157273482-00d16043-657b-448d-baed-df0b38fc279c.png)


#### Testing instructions

1. Go to WooCommerce Payments settings page (wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments)
2. Scroll to the area where payments methods are listed
3. You should see fee prices where prices have "From" prefix
4. For payments methods where base fee percentage doesn't exit, the lowest percentage fee should be displayed instead (iDeal might be a good example here, but it might not be in your case).
5. Mouse over the price should show tooltip or modal where fees are explained in detail (like it was before)


-------------------
